pages/basic-features/social-login.mdx

```diff
@@ -0,0 +1,80 @@
+import { Callout, Tabs, Tab } from "nextra-theme-docs";
+
+# Social Login
+
+This feature empowers you to seamlessly integrate social login functionality into your application, enhancing user experience and engagement. Craftable PRO supports various popular social platforms, including Facebook, GitHub, Twitter, Apple, Google, and Microsoft, allowing your users to sign in using their preferred social accounts effortlessly.
+The functionality is powered by the [Laravel Socialite](https://laravel.com/docs/socialite) package and its extensive range of providers.
+
+## Getting Started
+
+To enable social login in your application, follow these simple steps:
+
+<div className="steps-container">
+### Select which services you want to use and enable them in `config/craftable-pro.php`:
+```php filename="config/craftable-pro.php"
+   'social_login' => [
+        'allowed_services' => [
+            'microsoft' => false,
+            'github' => true,
+            'google' => true,
+            'twitter' => false,
+            'facebook' => false,
+            'apple' => false,
+        ],
+        'self_registration' => [
+            // define if users can self register into craftable pro interface
+            'enabled' => true,
+            // and if enabled, then which role(s) they should have assigned by default. Use role names here.
+            // It can be a string for one role or an array for multiple roles.
+            'default_role' => 'Administrator',
+            'allowed_domains' => ['craftable.pro'], // use * for allowing any domain
+        ]
+    ]
+```
+### If you enable social login self-registration, don't forget to also check the `allowed_domains`.
```

Comment:
I would expect more information about self-registration, at least mentioning that allowed domain means domain part of email.